cea/demand/thermal_loads.py

```diff
@@ -1148,12 +1148,12 @@ def get_temperatures(locator, prop_HVAC):
     :type prop_HVAC: Gdf
 
     Sample data (first 5 rows):
-                 Name type_cs type_ctrl type_dhw type_hs
-    0     B154862      T0        T1       T1      T1
-    1     B153604      T0        T1       T1      T1
-    2     B153831      T0        T1       T1      T1
-    3  B302022960      T0        T0       T0      T0
-    4  B302034063      T0        T0       T0      T0
+                 Name type_cs type_ctrl type_dhw type_hs MECH_VENT WIN_VENT HEAT_REC NIGHT_FLSH ECONOMIZER
```

Comment:
@JIMENOFONSECA: also it makes it almost impossible to study new technologies.
Imagine you want to equip all buildings with mechanical ventilation systems with a new type of heat exchanger. You just have to modify one key with the same value for all cases. In your case, you would have to modify every key of the heating system with an individual key based on the existing heating system (e.g. for building with T3 you have to change to T23, but for a building with T2, you would have to change to T54 to get the desired technical system configuration).